src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -0,0 +1,159 @@
+package frc.robot.subsystems;
+
+import static edu.wpi.first.wpilibj2.command.Commands.runOnce;
+
+import java.time.Instant;
+import java.util.function.DoubleSupplier;
+
+import org.littletonrobotics.junction.Logger;
+
+import com.chopshop166.chopshoplib.commands.SmartSubsystemBase;
+import com.chopshop166.chopshoplib.motors.SmartMotorController;
+import com.fasterxml.jackson.annotation.JacksonInject.Value;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.Commands;
+import frc.robot.maps.subsystems.ArmMap;
+import frc.robot.maps.subsystems.ArmMap.Data;
+
+public class Arm extends SmartSubsystemBase {
+
+    Data data;
+    ArmMap map;
+    private SmartMotorController extendMotor = new SmartMotorController();
+    private final double SOFT_MAX_DISTANCE = 400.0;
+    private final double SOFT_MIN_DISTANCE = 20.0;
+    PIDController pid = new PIDController(0.03, 0, 0);
```

Comment:
`private final`

Reply:
The PID constants should come from the Robot Map similar to `SOFT_MAX_DISTANCE`.